### PR TITLE
Fix #277: Added border to the columns in the log view.

### LINKED ISSUE #277: Give some space to the columns in the log view
Currently the columns in the log view are very closed to each other.

It would be good to give them some space.

Originally reported by @glensc in #268

### PR DESCRIPTION
**Pull request by souravbadami**
Added border to the columns in the log view. #277

### CODE REVIEW

File: anitya/templates/logs.html [hunk 1]
```diff
@@ -94,11 +94,11 @@
 
 </div>
 
-
-<table>
+<br>
```
Comment (pypingou) [inline anitya/templates/logs.html]: This line seems odd

Comment (souravbadami) [inline anitya/templates/logs.html]: Will it be appropriate to add margin instead of **br** to separate the above form the table ?

Comment (pypingou) [inline anitya/templates/logs.html]: Sounds like a better plan for me yes, but that should be added in the CSS directly not in the html

Comment (souravbadami) [inline anitya/templates/logs.html]: Done.

Comment (souravbadami) [inline anitya/templates/logs.html]: @pypingou -- Can you PTAL ?

Comment (pypingou) [inline anitya/templates/logs.html]: PTAL? I don't know this one :)

Comment (souravbadami) [inline anitya/templates/logs.html]: Please take a look :)

Comment (pypingou) [inline anitya/templates/logs.html]: Ah ok, cool :)

Change looks good I just wanted to test it locally before merging it :)

Comment (souravbadami) [inline anitya/templates/logs.html]: Sure, Thanks!

**Comment by pypingou** [comment]
Checked locally, looks cool

Thanks! :)

**Pull request merged**